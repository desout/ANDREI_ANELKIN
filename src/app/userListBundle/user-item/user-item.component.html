<div class="item">
  <div class="leftSideInfo">
    <span>{{user?.name}}</span><br>
    <span>{{user?.age}}</span><br>
    <span>{{user?.information}}</span>
  </div>
  <div class="rightSideInfo">
    <span>{{user?.dateOfBirth}}</span>
  </div>
</div>
