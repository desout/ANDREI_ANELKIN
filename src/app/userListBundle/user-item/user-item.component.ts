import {Component, Input, OnInit} from '@angular/core';
import {User} from '../../../../server/server/User';

@Component({
  selector: 'app-user-item',
  templateUrl: './user-item.component.html',
  styleUrls: ['./user-item.component.scss']
})
export class UserItemComponent implements OnInit {
  @Input() user: User;
  constructor() { }

  ngOnInit() {
  }

}
