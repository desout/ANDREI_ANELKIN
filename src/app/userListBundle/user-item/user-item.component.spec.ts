import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { UserItemComponent } from './user-item.component';
import {NO_ERRORS_SCHEMA} from '@angular/core';
import {TestModule} from '../../Test.module';

describe('UserItemComponent', () => {
  let component: UserItemComponent;
  let fixture: ComponentFixture<UserItemComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [TestModule],
      declarations: [  ],
      schemas: [NO_ERRORS_SCHEMA]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserItemComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
