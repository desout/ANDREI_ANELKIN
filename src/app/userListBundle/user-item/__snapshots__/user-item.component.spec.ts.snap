// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserItemComponent should create 1`] = `
<div
  id="root0"
  ng-version="7.0.3"
>
  <div
    class="item"
  >
    <div
      class="leftSideInfo"
    >
      <span>
        
      </span>
      <br />
      <span>
        
      </span>
      <br />
      <span>
        
      </span>
    </div>
    <div
      class="rightSideInfo"
    >
      <span>
        
      </span>
    </div>
  </div>
</div>
`;
